packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -283,6 +283,27 @@ def test_3dt_units(cluster):
         )
 
 
+@retrying.retry(wait_fixed=2000, stop_max_delay=LATENCY*1000)
+def test_3dt_units_health(cluster):
```

Comment:
I'd suggest taking "3dt" out of the name so that people don't think 3dt itself is the thing under test here. That should hopefully reduce the pings you get for other people's broken stuff :)